charts/sourcegraph-appliance/values.yaml

```diff
@@ -57,8 +57,7 @@ ingress:
   enabled: false
   className: ""
   annotations:
-    {}
-    # kubernetes.io/ingress.class: nginx
+    kubernetes.io/ingress.class: alb
```

Comment:
y tho

Reply:
this is still a mystery AFAIK

Reply:
I think I forgot to change branch / didn't clean up my changes